drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1686,15 +1686,13 @@ function DragAndDropBlock(runtime, element, configuration) {
         $.post(url, JSON.stringify(data), 'json')
             .done(function(data){
                 state.items[item_id].submitting_location = false;
-                // In standard mode we immediately return item to the bank if dropped on wrong zone.
+                // In standard mode we return wrongly dropped item to the bank when feedback popup is closed.
                 // In assessment mode we leave it in the chosen zone until explicit answer submission.
                 if (configuration.mode === DragAndDropBlock.STANDARD_MODE) {
```

Comment:
This feature would also be useful in assessment mode, but I think this is out of scope for this change?

Reply:
yes there would be separate ticket for assessment mode, if needed.